build

```
@@ -4,7 +4,7 @@ NAME="coreos-metadata"
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/${NAME}"
 VERSION=$(git describe --dirty)
-GLDFLAGS="-X main.version \"${VERSION}\""
+GLDFLAGS="-X main.version=\"${VERSION}\""
```

Wow, you've got to be fucking kidding me. It silently fails. Go strikes again!